WalletWasabi/Crypto/ZeroKnowledge/ProofSystem.cs

```diff
@@ -40,5 +41,19 @@ public static Statement IssuerParameters(CoordinatorParameters iparams, MAC mac,
 				{ Generators.GV - iparams.I, O,             O,              Generators.Gx0, Generators.Gx1, Generators.Ga },
 				{ iparams.Cw,                Generators.Gw, Generators.Gwp, O,              O,              O },
 			});
+
+		public static Knowledge ShowCredential(CredentialPresentation c, Scalar z, Scalar t, Scalar a, Scalar r, CoordinatorParameters iparams)
```

Comment:
oh wait, codescene is still complaining about this, this is why i proposed already adding `Credential` here because it can be passed to the proof system helpers much like `CoordinatorSecretKey` is for the issuer params